platforms/web/scripts/i18next/parse-translations.ts

```diff
@@ -0,0 +1,66 @@
+import { existsSync, readdirSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { execSync } from 'node:child_process';
+
+import prettier from 'prettier';
+
+const i18nextParserConfig = join(process.cwd(), 'i18next-parser.config.js');
+const platformGlob = join(process.cwd(), 'src/**/*.{ts,tsx}');
+
+type WorkspaceEntry = {
+  location: string;
+  workspaceDependencies: string[];
+};
+
+type WorkspaceInfo = Record<string, WorkspaceEntry>;
+
+function syncTranslations() {
+  const result = execSync('yarn workspaces --json info').toString('utf-8');
+  const info = JSON.parse(result);
+  const data = JSON.parse(info.data) as WorkspaceInfo;
+
+  const dependencies = data['@jwp/ott-web']?.workspaceDependencies || [];
+
+  const globs = dependencies
+    .map((dependency) => data[dependency])
+    .filter((entry) => entry && existsSync(join('../..', entry.location, 'src')))
```

Comment:
If we are going to use the assumption that dependencies should have a /src directory, I would prefer we throw an error when one doesn't exist instead of just filtering them out. Otherwise it may be hard to debug later down the road.

Reply:
Hmmm interesting point. The workspaces info command returns all workspace packages including the config packages. So, throwing an error would be a bit too strict IMO.

We could log which workspace packages are skipped, but this will clutter the output because we know which ones should be ignored.

In theory, we could check if the `i18next` dependency is listed in the workspace package and only include it then...

PS: This will be solved when we parse the exact files needed for extraction (see below).